SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -323,6 +323,15 @@ TArray<UClass*> GetAllSupportedClasses()
 			}
 		}
 
+		for (TFieldIterator<UFunction> FunctionIt(*ClassIt); FunctionIt; ++FunctionIt)
```

Comment:
nit: This will unnecessarily get called if SupportedClass was already found.

Reply:
More than a nit. Please fix.